src/main/java/com/serjltt/moshi/adapters/Wrapped.java

```diff
@@ -64,22 +64,37 @@
 @Documented
 @JsonQualifier
 @Retention(RetentionPolicy.RUNTIME)
-@Target({ ElementType.FIELD, ElementType.METHOD, ElementType.PARAMETER })
+@Target({ElementType.FIELD, ElementType.METHOD, ElementType.PARAMETER})
 public @interface Wrapped {
   /** The path to the wrapped json value. */
-  String[] value();
+  String[] path();
+
+  /**
+   * Indicates if the adapter should fail if the json object was not found at the indicated path.
```

Comment:
maybe: Indicates if the adapter should fail **when** the json object was not found at the indicated path